packages/styling/src/interfaces/ITheme.ts

```diff
@@ -35,12 +37,20 @@ export interface ITheme {
    * Avoid using it until it is finalized.
    */
   spacing: ISpacing;
+
+  /**
+ * @internal
+ * The schemes property is still in an experimental phase. The intent is to have it work
+ * in conjunction with new 'schemes' prop that any component making use of Foundation can use.
+ * Alternative themes that can be referred to by name.
+ */
+  schemes?: { [P in IScheme]?: ITheme };
```

Comment:
At the end of this, IScheme should probably be what ITheme is today, and then:

ITheme extends IScheme {
  schemes?: { blah };
}

To avoid recursive issues.

Reply:
Hmm, are there any backwards compatibility concerns with this approach?

Reply:
I don't see a backwards compat issue.

Reply:
I am not sure if it matter that we have 2 interfaces vs one. It doesn't hurt, but I'm not totally sure of the use case.

Reply:
I'm a bit torn on this. I don't want `ITheme` to have `schemes` at all since schemes don't take effect everywhere. (Say if themes are used in `scopedSettings`, for example.) It's just that highest theme definition level (contextual or static) that will actually use themes. I feel like we need something like `IThemeContext` that's the only structure with a `schemes` property.